core/rawdb/database.go

```diff
@@ -41,7 +41,20 @@ type freezerdb struct {
 	ancientRoot string
 	ethdb.KeyValueStore
 	ethdb.AncientStore
-	diffStore ethdb.KeyValueStore
+	diffStore  ethdb.KeyValueStore
+	blockStore ethdb.KeyValueStore
```

Comment:
Considering that the blockstore is composed of kvdb and ancientdb, the blockStore should changed from `KeyValueStore` to `DataBase`